File: relactional.java
import java.util.Scanner;
public class relactional
{   
    int num,den;
    Scanner reading = new Scanner(System.in);
    public relactional()
    {
        num = 0 ;
        den = 1 ;
    }
    public relactional( int x , int y)
    {
        if ( y == 0 )
        {
            System.out.println("Enter Correct data.");
            System.exit(0);
        }
        num = x ;
        den = y ;
    }
        void input()
        {
            System.out.println("Enter value of num. :: ");
            num = reading.nextInt();
            System.out.println("Enter value of den. :: ");
            den = reading.nextInt();
        }
        void output()
        {
            System.out.println( num + "/" + den );
        }
        void munt( relactional a,relactional b )
        {
            num = a.num*b.num;
            den = a.den*b.den;
        }
    public static void main( String [] args )
    {
        relactional a = new relactional();
        relactional b = new relactional();
        relactional c = new relactional();
        a.input();
        b.input();
        c.munt(a,b);
        c.output();
    }
}

File: relactional.py
class relactional:
    def __init__ (self,a=0,b=1):
        self.num,self.den = a,b
    def __str__(self):
       return "%d/%d"%(self.num,self.den)
    def inputd(self):
       self.num = int(input("Enter value of num : "))
       self.den = int(input("Enter value of den : "))
    def add(self,a):
        z = relactional();
        z.num = self.num*b.den + b.num*self.den
        z.den = self.den*b.den
        return z
    def sub(self,a):
        z = relactional();
        z.num = self.num*b.den - b.num*self.den
        z.den = self.den*b.den
        return z
a = relactional(2,2)
b = relactional()
b.inputd()
c = a.sub(b)
print(c)
c = a.add(b)
print(c)